packages/hooks/src/useWalletTransactions.ts

```diff
@@ -1,7 +1,6 @@
 import { useEffect, useMemo, useState } from 'react';
 import { useStore } from '@deriv/stores';
-import { usePaginatedFetch } from '@deriv/api';
-import useCurrencyConfig from './useCurrencyConfig';
+import { usePaginatedFetch, useCurrencyConfig } from '@deriv/api';
```

Comment:
refactored test because of replacing `useCurrencyConfig` from `@deriv/hooks` to `useCurrencyConfig` from `@deriv/api`